CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Braintree Android SDK Release Notes
 
+## unreleased
+
+* PayPalNativeCheckout
+  * Bump native-checkout version to release `1.2.1`
```

Comment:
```suggestion
  * Bump native-checkout version to release `1.2.1`
  * Upgraded the data-collector SDK to version 3.21.0 which made updates to Device Data collection related to Google Play's User Data Policy. For more info read the [release notes](https://github.com/paypal/android-checkout-sdk/releases/tag/v1.2.1)
```

Reply:
@hollabaq86 @jaxdesmarais how about this?

Reply:
That sounds good to me!